spec/factories/package_categories.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :package_category do
+    name_en    "MyString"
+    name_zh_tw "MyString"
```

Comment:
@swatijadhav re comment above, we can wire this up to the real data using yaml files and factory_girl's ```generate``` method. e.g. https://github.com/crossroads/api.goodcity/blob/master/spec/factories/territories.rb#L5